### PR TITLE
Update gsheets.md

### PR DESCRIPTION
**Pull request by rpourzand**
Google Sheets updates

Merge status: Do not merge this PR. Waiting on PM to give merge go ahead as they're still working through things. ✅ PM approved and ok to merge

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md [hunk 1]
```diff
@@ -53,18 +53,22 @@ The Google Sheets **Query Builder** custom menu has the following capabilities:
 | Menu items    | Description                                           |
 |---------------|-------------------------------------------------------|
 | Metrics       | Search and select metrics.                             |
-| Group By      | Search and select dimensions or entities to group by. Dimensions are grouped by the entity of the semantic model they come from. |
-| Time Range    | Quickly select time ranges to look at the data, which applies to the main time series for the metrics (metric time). You can still optionally use `where` to filter time ranges|
+| Group By      | Search and select dimensions or entities to group by. Dimensions are grouped by the entity of the semantic model they come from. You may choose dimensions on their own without metrics |
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/gsheets.md]:
```suggestion
| Group By      | Search and select dimensions or entities to group by. Dimensions are grouped by the entity of the semantic model they come from. You may choose dimensions on their own without metrics. |
```

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md [hunk 2]
```diff
@@ -53,18 +53,22 @@ The Google Sheets **Query Builder** custom menu has the following capabilities:
...
 | Limit         | Set a limit for the rows of your output.               |
 
+
+
 **Filtering data**
 
 To use the filter functionality, choose the [dimension](docs/build/dimensions) you want to filter by and select the operation you want to filter on.
-- For categorical dimensions, type in the dimension value you want to filter by (no quotes needed) and press enter. 
-- Continue adding additional filters as needed with AND and OR. If it's a time dimension, choose the operator and select from the calendar.
+- For categorical dimensions,you can type a value in to search, or select from a populated list. For entities, you must type the value you are looking for as we do not load all of them given the large number of values.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/gsheets.md]:
```suggestion
- For categorical dimensions, you can type a value into search or select from a populated list. For entities, you must type the value you are looking for as we do not load all of them given the large number of values.
```

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md [hunk 3]
```diff
@@ -53,18 +53,22 @@ The Google Sheets **Query Builder** custom menu has the following capabilities:
...
+
 **Filtering data**
 
 To use the filter functionality, choose the [dimension](docs/build/dimensions) you want to filter by and select the operation you want to filter on.
-- For categorical dimensions, type in the dimension value you want to filter by (no quotes needed) and press enter. 
-- Continue adding additional filters as needed with AND and OR. If it's a time dimension, choose the operator and select from the calendar.
+- For categorical dimensions, you can type a value into search or select from a populated list. For entities, you must type the value you are looking for as we do not load all of them given the large number of values.
+- Continue adding additional filters as needed with AND and OR.
+
+For time dimensions, you can filter on the primary time dimension using time tange selector which offers presets, or use "custom" for additional options. You can also use the "Where" option to filter on time dimensions that are not the primary time series for the metrics.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/gsheets.md]:
```suggestion
For time dimensions, you can filter on the primary time dimension using the time range selector which offers presets, or use "custom" for additional options. You can also use the "Where" option to filter on time dimensions that are not the primary time series for the metrics.
```

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md [hunk 4]
```diff
@@ -53,18 +53,22 @@ The Google Sheets **Query Builder** custom menu has the following capabilities:
...
+
 **Filtering data**
 
 To use the filter functionality, choose the [dimension](docs/build/dimensions) you want to filter by and select the operation you want to filter on.
-- For categorical dimensions, type in the dimension value you want to filter by (no quotes needed) and press enter. 
-- Continue adding additional filters as needed with AND and OR. If it's a time dimension, choose the operator and select from the calendar.
+- For categorical dimensions, you can type a value into search or select from a populated list. For entities, you must type the value you are looking for as we do not load all of them given the large number of values.
+- Continue adding additional filters as needed with AND and OR.
+
+For time dimensions, you can filter on the primary time dimension using the time range selector which offers presets, or use "custom" for additional options. You can also use the "Where" option to filter on time dimensions that are not the primary time series for the metrics.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/gsheets.md]:
```suggestion
For time dimensions, you can use the time range selector to filter on the primary time dimension with preset or custom options. For non-primary time dimensions, use the "Where" option to apply filters.
```

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
 ---
-title: "Google Sheets (beta)"
```
Comment (DevonFulcher) [inline website/docs/docs/use-dbt-semantic-layer/gsheets.md:2]: 🚀

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md [hunk 6]
```diff
@@ -1,8 +1,8 @@
...
+title: "Google Sheets"
 description: "Integrate with Google Sheets to query your metrics in a spreadsheet."
 tags: [Semantic Layer]
-sidebar_label: "Google Sheets (Preview)"
+sidebar_label: "Google Sheets"
 ---
 
 # Google Sheets <Lifecycle status='preview'/>
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/gsheets.md]: removing the preview tag as well
Screenshot 2024-04-29 at 17 06 39


```suggestion
```

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md [hunk 7]
```diff
@@ -1,11 +1,10 @@
...
+title: "Google Sheets"
 description: "Integrate with Google Sheets to query your metrics in a spreadsheet."
 tags: [Semantic Layer]
-sidebar_label: "Google Sheets (Preview)"
+sidebar_label: "Google Sheets"
 ---
 
-# Google Sheets <Lifecycle status='preview'/>
 
 :::info Preview functionality
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/gsheets.md]:
```suggestion
:::info Preview functionality
```

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md [hunk 8]
```diff
@@ -1,11 +1,10 @@
...
 Google Sheets integration with the dbt Semantic Layer is a [Preview](/docs/dbt-versions/product-lifecycles#dbt-cloud) feature.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/gsheets.md]: we should prob remove this callout too right @rpourzand ?

File: website/snippets/_sl-partner-links.md [hunk 9]
```diff
@@ -11,7 +11,7 @@ The following tools integrate with the dbt Semantic Layer:
     icon="tableau-software"/>
 
   <Card
-    title="Google Sheets (Preview)"
+    title="Google Sheets"
```
Comment (mirnawong1) [inline website/snippets/_sl-partner-links.md:14]: removed the 'preview' mention from the main avail integrations card parge too @rpourzand 

Screenshot 2024-04-29 at 17 05 31

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md (hunk 7 above)
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/gsheets.md]:
```suggestion
```

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md [hunk 10]
```diff
@@ -53,18 +52,22 @@ The Google Sheets **Query Builder** custom menu has the following capabilities:
...
+
 **Filtering data**
 
 To use the filter functionality, choose the [dimension](docs/build/dimensions) you want to filter by and select the operation you want to filter on.
-- For categorical dimensions, type in the dimension value you want to filter by (no quotes needed) and press enter. 
-- Continue adding additional filters as needed with AND and OR. If it's a time dimension, choose the operator and select from the calendar.
+- For categorical dimensions, you can type a value into search or select from a populated list. For entities, you must type the value you are looking for as we do not load all of them given the large number of values.
+- Continue adding additional filters as needed with AND and OR.
+
+For time dimensions, you can use the time range selector to filter on the primary time dimension with preset or custom options. For non-primary time dimensions, use the "Where" option to apply filters.
```
Comment (DevonFulcher) [inline website/docs/docs/use-dbt-semantic-layer/gsheets.md]: By `non-primary time dimensions`, do you mean non-metric time dimensions? The time range only works for metric time.

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md (hunk 8 above)
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/gsheets.md]:
```suggestion
```

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md [hunk 11]
```diff
@@ -1,15 +1,10 @@
...
+title: "Google Sheets"
 description: "Integrate with Google Sheets to query your metrics in a spreadsheet."
 tags: [Semantic Layer]
-sidebar_label: "Google Sheets (Preview)"
+sidebar_label: "Google Sheets"
 ---
 
-# Google Sheets <Lifecycle status='preview'/>
-
-:::info Preview functionality
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/gsheets.md:10]: removed this @rpourzand for consistency

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md (hunk 10 above)
Comment (mirnawong1) [reply website/docs/docs/use-dbt-semantic-layer/gsheets.md]: i changed it and i think the original was '...to filter on time dimensions that are not the primary time series for the metrics.' maybe i should add 'series for metrics' at the end

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md [hunk 12]
```diff
@@ -53,18 +48,22 @@ The Google Sheets **Query Builder** custom menu has the following capabilities:
...
+
 **Filtering data**
 
 To use the filter functionality, choose the [dimension](docs/build/dimensions) you want to filter by and select the operation you want to filter on.
-- For categorical dimensions, type in the dimension value you want to filter by (no quotes needed) and press enter. 
-- Continue adding additional filters as needed with AND and OR. If it's a time dimension, choose the operator and select from the calendar.
+- For categorical dimensions, you can type a value into search or select from a populated list. For entities, you must type the value you are looking for as we do not load all of them given the large number of values.
+- Continue adding additional filters as needed with AND and OR.
+
+For time dimensions, you can use the time range selector to filter on the primary time dimension with preset or custom options. For non-primary time dimensions, use the "Where" option to apply filters.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/gsheets.md]: it sounds a little awkward so am going to wait for @rpourzand 

```suggestion
For time dimensions, you can use the time range selector to filter on the primary time dimension with preset or custom options. For non-primary time dimension series for metrics, use the "Where" option to apply filters.
```

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md (hunk 11 above)
Comment (rpourzand) [reply website/docs/docs/use-dbt-semantic-layer/gsheets.md:10]: Thank you!

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md (hunk 8 above)
Comment (rpourzand) [reply website/docs/docs/use-dbt-semantic-layer/gsheets.md]: Yep! good call

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md (hunk 10 above)
Comment (rpourzand) [reply website/docs/docs/use-dbt-semantic-layer/gsheets.md]: Correct. Anything that's not metric_time!

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md (hunk 12 above)
Comment (mirnawong1) [reply website/docs/docs/use-dbt-semantic-layer/gsheets.md]: should we say 'For non-metric time dimensions, use the "where"...'?

Comment (rpourzand) [reply website/docs/docs/use-dbt-semantic-layer/gsheets.md]: Ok I tweaked a bit! let me know your thoughts

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md [hunk 13]
```diff
@@ -53,18 +48,22 @@ The Google Sheets **Query Builder** custom menu has the following capabilities:
...
+
 **Filtering data**
 
 To use the filter functionality, choose the [dimension](docs/build/dimensions) you want to filter by and select the operation you want to filter on.
-- For categorical dimensions, type in the dimension value you want to filter by (no quotes needed) and press enter. 
-- Continue adding additional filters as needed with AND and OR. If it's a time dimension, choose the operator and select from the calendar.
+- For categorical dimensions, you can type a value into search or select from a populated list. For entities, you must type the value you are looking for as we do not load all of them given the large number of values.
+- Continue adding additional filters as needed with AND and OR.
+
+For time dimensions, you can use the time range selector to filter on presets or custom options. The time range selector applies only to the primary time dimension (metric time). For all other  time dimensions that are not metric time, you can use the "Where" option to apply filters.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/gsheets.md]:
```suggestion
For time dimensions, you can use the time range selector to filter on presets or custom options. The time range selector applies only to the primary time dimension (`metric_time`). For all other time dimensions that aren't `metric_time`, you can use the "Where" option to apply filters.
```

**Review by mirnawong1 — APPROVED** [review]
lgtm! do you want me to merge @rpourzand @DevonFulcher or hang tight?

**Comment by rpourzand** [comment]
@mirnawong1  let's wait a bit to launch. We're still fixing some bugs. I'll give you the go ahead

**Comment by rpourzand** [comment]
@mirnawong1  this can be released first thing tomorrow!

**Pull request merged**